packages/contextual-help/src/ContextualHelp.ts

```diff
@@ -0,0 +1,205 @@
+    SlottableRequestEvent,
+} from '@spectrum-web-components/overlay/src/slottable-request-event.js';
+import type { Placement } from '@spectrum-web-components/overlay/src/overlay-types.js';
+import {
+    IS_MOBILE,
+    MatchMediaController,
+} from '@spectrum-web-components/reactive-controllers/src/MatchMedia.js';
+
+import styles from './contextual-help.css.js';
+
+/**
+ * Spectrum Contextual help provides additional information about
+ * the state of either an adjacent component or an entire view.
+ * @element sp-contextual-help
+ *
+ * @slot Text content to display in the popover
+ * @slot link - link to additional informations
+ */
+export class ContextualHelp extends SpectrumElement {
+    protected isMobile = new MatchMediaController(this, IS_MOBILE);
+
+    public static override get styles(): CSSResultArray {
+        return [styles];
+    }
+
+    /**
+     * Optional title to be displayed inside the popover.
+     * @param {String} headline
+     */
+    @property()
+    public headline?: string;
+
+    /**
+     * Provides an accessible name for the action button trigger.
+     * @param {String} label
+     */
+    @property()
+    public label?: string;
+
+    /**
+     * The `variant` property applies specific styling on the action button trigger.
+     * @param {String} variant
+     */
+    @property()
+    public variant: 'info' | 'help' = 'info';
+
+    /**
+     * @type {"top" | "top-start" | "top-end" | "right" | "right-start" | "right-end" | "bottom" | "bottom-start" | "bottom-end" | "left" | "left-start" | "left-end"}
+     * @attr
+     */
+    @property({ reflect: true })
+    public placement?: Placement = 'bottom-start';
+
+    /**
+     * The `offset` property accepts either a single number, to
+     * define the offset of the Popover along the main axis from
+     * the action button, or 2-tuple, to define the offset along the
+     * main axis and the cross axis.
+     */
+    @property({ type: Number })
+    public offset: number | [number, number] = 0;
+
+    @state()
+    overlayOpen = false;
+
+    private get buttonAriaLabel(): string {
+        if (this.label) {
+            return this.label;
+        } else {
+            if (this.variant === 'help') {
+                return 'Help';
+            }
+            return 'Informations';
+        }
+    }
+
+    private renderPopover(event: SlottableRequestEvent): void {
+        event.stopPropagation();
+
+        if (event.data === removeSlottableRequest) {
+            render(undefined, event.target as HTMLElement);
+            return;
+        }
+
+        import('@spectrum-web-components/popover/sp-popover.js');
+
+        const template = html`
+            <sp-popover class="popover">
+                <section>
+                    ${this.headline &&
+                    html`
+                        <h2 class="heading">${this.headline}</h2>
+                    `}
+                    <slot></slot>
+                    <div class="link">
```

Comment:
Styling a wrapper means that even when a consumer does not provide a link the padding/margin managing it will be present. Can we point rules for this onto the slotted element, which will only apply if one is present?